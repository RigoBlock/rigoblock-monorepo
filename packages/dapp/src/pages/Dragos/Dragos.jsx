import BaseTemplate from '../../components/templates/BaseTemplate'
import React, { Component } from 'react'

class Dragos extends Component {
  render() {
    return (
      <BaseTemplate>
        <h1>Dragos</h1>
      </BaseTemplate>
    )
  }
}

export default Dragos
