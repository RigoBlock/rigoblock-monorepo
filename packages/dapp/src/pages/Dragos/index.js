export { default } from './Dragos.jsx'
