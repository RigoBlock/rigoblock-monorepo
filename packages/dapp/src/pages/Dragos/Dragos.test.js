import { shallow } from 'enzyme'
import Dragos from './Dragos.jsx'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('Dragos page', () => {
  it('renders correctly', () => {
    expect(toJson(shallow(<Dragos />))).toMatchSnapshot()
  })
})
