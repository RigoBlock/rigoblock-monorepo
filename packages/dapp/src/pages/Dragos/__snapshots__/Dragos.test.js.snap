// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dragos page renders correctly 1`] = `
<Connect(BaseTemplate)>
  <h1>
    Dragos
  </h1>
</Connect(BaseTemplate)>
`;
